build.gradle

```diff
@@ -104,29 +22,23 @@ dependencies {
     implementation 'org.springframework.data:spring-data-redis'
     implementation 'redis.clients:jedis:3.3.0'
 
-    implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:2.2.1.RELEASE")
+    implementation 'org.springframework.cloud:spring-cloud-starter-aws-messaging:2.2.2.RELEASE'
     implementation 'org.springframework:spring-jms:5.2.6.RELEASE'
     implementation 'com.amazonaws:amazon-sqs-java-messaging-lib:1.0.8'
 
     implementation("org.springframework.boot:spring-boot-starter-security")
-    implementation("org.springframework.security.oauth:spring-security-oauth2:2.4.1.RELEASE")
-    implementation("org.springframework.security:spring-security-jwt:1.1.0.RELEASE")
-
-    implementation 'org.apache.camel:camel-spring-boot:2.25.1'
-    implementation 'org.apache.camel:camel-csv:2.25.1'
-    implementation ('org.apache.camel:camel-aws:2.25.1') {
-        exclude group: 'com.amazonaws', module: 'aws-java-sdk-sagemaker'
-        exclude group: 'com.amazonaws', module: 'aws-java-sdk-sagemakerruntime'
-        exclude group: 'com.amazonaws', module: 'aws-java-sdk-opsworkscm'
-        exclude group: 'com.amazonaws', module: 'aws-java-sdk-eks'
-    }
-    implementation 'net.javacrumbs.shedlock:shedlock-spring:4.10.1'
-    implementation 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.10.1'
-
-    implementation 'net.logstash.logback:logstash-logback-encoder:6.3'
-    implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:2.6.0'
-    implementation 'com.microsoft.azure:applicationinsights-logging-logback:2.6.0'
-    agentDeps 'com.microsoft.azure:applicationinsights-agent:2.6.0'
+    implementation("org.springframework.security.oauth:spring-security-oauth2:2.5.0.RELEASE")
+    implementation("org.springframework.security:spring-security-jwt:1.1.1.RELEASE")
+
+    compile group: 'org.hibernate.validator', name: 'hibernate-validator', version: '6.1.5.Final'
```

Comment:
`compile` deprecated - replaced with `implementation`.
Should really be
```
    implementation 'org.hibernate.validator:hibernate-validator:6.1.5.Final'
```
do you really need it at all though - isn't it included with `gradle-spring-boot`?

Reply:
yup, its ok without it now